src/panels/MinimalPanel.vue

```diff
@@ -10,42 +10,51 @@
       </button>
     </div>
     <div v-show="!localMinimized" class="card card-flex" @mouseover="onHeaderHover = true" @mouseleave="onHeaderHover = false">
-      <div :class="['header-wrapper', { 'header-wrapper-bottom': localExpanded, 'header-toggle': isExpandableCard }]"
+      <div :class="['header-wrapper', { 'header-wrapper-bottom': isHeaderAtBottom, 'header-toggle': isExpandableCard }]"
            @click.prevent.stop="isExpandableCard && toggle()">
-        <span v-show="!localExpanded" ref="headerWrapper"
-              :class="['card-title', 'card-title-transparent', { 'card-title-opaque': onHeaderHover, 'ellipses': !hasHeaderBool }]">
-          <slot name="header">
-            <span class="card-title-inline"><slot name="_header"></slot></span>
-            <span v-show="showDownSwitch" aria-hidden="true"
-                  class="minimal-button glyphicon glyphicon-menu-down minimal-menu-down"></span>
-          </slot>
-        </span>
-        <div :class="['button-wrapper', { 'button-wrapper-expanded': localExpanded, 'button-wrapper-visible': onHeaderHover }]">
+        <transition name="title">
+          <span v-show="!isHeaderAtBottom" ref="headerWrapper"
+                :class="['card-title', 'card-title-transparent', { 'card-title-opaque': onHeaderHover, 'ellipses': !hasHeaderBool }]">
+            <slot name="header">
+              <span class="card-title-inline"><slot name="_header"></slot></span>
+              <span v-show="showDownSwitch" aria-hidden="true"
+                    class="minimal-button glyphicon glyphicon-menu-down minimal-menu-down"></span>
+            </slot>
+          </span>
+        </transition>
+        <div :class="['button-wrapper', { 'button-wrapper-expanded': isHeaderAtBottom, 'button-wrapper-visible': onHeaderHover }]">
           <slot name="button">
             <button v-show="!noCloseBool" class="minimal-button" type="button" @click.stop="close()">
               <span class="glyphicon glyphicon-remove minimal-close-button" aria-hidden="true"></span>
             </button>
             <button v-show="popupUrl !== null" class="minimal-button" type="button" @click.stop="openPopup()">
               <span class="glyphicon glyphicon-new-window minimal-popup-button" aria-hidden="true"></span>
             </button>
-            <button v-show="localExpanded && isExpandableCard && bottomSwitchBool" class="minimal-button" type="button"
-                    @click.prevent.stop="collapseThenScrollIntoViewIfNeeded()">
+            <button v-show="isHeaderAtBottom && isExpandableCard && bottomSwitchBool" class="minimal-button" type="button">
```

Comment:
Simple removal of `collapseThenScrollIntoViewIfNeeded()` that wasn't working before